### PR TITLE
Support for .NET Standard

### PR DESCRIPTION
**Pull request by vividos**
Xamarin app development in Visual Studio 2017 is going to adapt .NET Standard libraries more and more. Most of the Xamarin.Plugin libraries that I'm using in my project were already ported to use .NET Standard (Geolocator, External Maps, Permissions, etc.). FilePicker is still using PCL, so I set out to port it to .NET Standard.

Here is a pull request with my porting efforts. Please check each commit and accept it, request modifications or cherry-pick changes as you like.

Unfortunately when developing with Visual Studio 2017, Microsoft dropped support for Windows 8.1 and Windows Phone 8.1, so I also had to remove it from the .sln file. The projects are still there.
I also moved and modified the .nuspec file, as in the current location no package could be built. I set the version to "2.0.0-beta", which should allow publishing the package to NuGet.org without disturbing users of previous versions.

The projects build without errors and the "nuget pack" generates a NuGet package that works in my Android-only project. As the platform-dependent projects weren't touched, I expect that other platforms work OK, too.

Finally, maybe this Pull Request should go into a "2.0" or "net standard" branch, since maybe more 1.3.x or 1.4.x versions are needed. I hope my contribution is useful for someone!

Note: This is adapted from the pull request on ArtjomP's fork of the FilePicker plugin: https://github.com/ArtjomP/FilePicker-Plugin-for-Xamarin-and-Windows/pull/4

### CODE REVIEW

**Comment by jfversluis** [comment]
Wow this looks great! And dropping Windows phone was on the list anyway So dont sorry about that. I’ll take a good look at this tomorrow, Thanks for your efforts!

File: src/Plugin.FilePicker/Plugin.FilePicker/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -26,5 +26,5 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
-[assembly: AssemblyVersion("1.3.0.0")]
-[assembly: AssemblyFileVersion("1.3.0.0")]
+[assembly: AssemblyVersion("2.0.0.0")]
```
Comment (jfversluis) [inline src/Plugin.FilePicker/Plugin.FilePicker/Properties/AssemblyInfo.cs:29]: All the version number changes are nice, but not necessary, this is typically done while building. No need to change it or anything, just wanted to let you know :)

**Review by jfversluis — COMMENTED** [review]
Looks good overall. To keep the repo clean, could you also delete the projects from disk that you have removed from the solution? They don't have any use and are in source control so no need to still keep them around here.

**Comment by vividos** [comment]
I removed the old projects, as you suggested, commit b6fb674.

**Comment by jfversluis** [comment]
Great! Thanks!

**Comment by ask-himanshu** [comment]
Can anyone tell me how to resolve this issue as I am not able to use this IOS device

**Comment by jfversluis** [comment]
I'm sorry but I don't see what issue you are referring to by this PR. Please open an issue and fill out the template or raise a question on StackOverflow if you need help with consuming this library.

**Comment by ask-himanshu** [comment]
System.NotImplementedException: This functionality is not implemented in the portable version of this assembly

**Comment by vividos** [comment]
@ask-himanshu please read the README for this project, the exact issue is described in the troubleshooting section. Also check the sample project if that works on iOS and compare with your project. Thanks!

**Pull request merged**